src/compose/ComposeBox.js

```diff
@@ -131,12 +131,15 @@ class ComposeBox extends PureComponent<Props, State> {
   };
 
   handleMessageFocus = () => {
+    const { topic } = this.state;
     const { lastMessageTopic } = this.props;
-    this.setState(({ topic }) => ({
+    this.setState({
       isMessageFocused: true,
       isMenuExpanded: false,
-      topic: topic || lastMessageTopic,
-    }));
+    });
+    setTimeout(() => {
+      this.handleTopicChange(topic || lastMessageTopic);
+    }, 200);
```

Comment:
Your commit message explains:

> To ensure the component is shown, wait a 'safe' time of 200ms.

This is helpful, but I'm not sure which component you mean: the ComposeBox itself, or the message input, or something else?

And why is this `setState` specifically (that's all `handleTopicChange` does, is call `setState`) a problem if that component isn't shown yet, while the `setState` just above this isn't?

Reply:
Oh, also: this explanation would be good to have in a comment in the code (like the ones on other setTimeouts in this file, in fact). That's because it's something that the reader might be surprised about just from looking at the code as it is, not only if they compare it to a previous version.